### ISSUE TITLE
Error on macOS + Python 3

### ISSUE THREAD

**Issue by mikeckennedy** [np-8/wakepy#11]
Hey! Trying this on macOS and Python 3.9 I got:

```python
Traceback (most recent call last):
  File "/Users/michaelkennedy/Desktop/wakepy/example_app/macos/macos_app.py", line 18, in <module>
    wakepy.unset_keepawake()
  File "/Users/michaelkennedy/Desktop/wakepy/wakepy/_darwin.py", line 38, in unset_keepawake
    _process.stdin.write(BREAK)
TypeError: a bytes-like object is required, not 'str'
```

I'll submit a PR to fix this in a moment.

**Comment by np-8** [comment]
Yeah I agree that the `Popen.terminate()` seems cleaner way to terminate the process. I've merged the PR and published the change with wakepy 0.4.3. on PyPi. Thanks for your contribution!

**Comment by mikeckennedy** [comment]
You’re welcome!

Best,
Michael

Sent via Superhuman iOS ***@***.***>
